go.mod

```diff
@@ -5,11 +5,11 @@ go 1.13
 require (
 	github.com/c2h5oh/datasize v0.0.0-20171227191756-4eba002a5eae
 	github.com/die-net/lrucache v0.0.0-20181227122439-19a39ef22a11
-	github.com/google/go-github/v30 v30.1.0
+	github.com/google/go-github/v31 v31.0.0
 	github.com/gregjones/httpcache v0.0.0-20190611155906-901d90724c79
 	github.com/inconshreveable/mousetrap v1.0.0 // indirect
 	github.com/palantir/go-baseapp v0.2.0
-	github.com/palantir/go-githubapp v0.2.0
+	github.com/palantir/go-githubapp v0.2.1-0.20200423005854-8924ef0f1e86
```

Comment:
Temporarily using a branch dependency because I'd like to validate async dispatch in a real application before tagging a `go-githubapp` release.